third_party/terraform/resources/resource_compute_instance.go

```diff
@@ -510,6 +510,37 @@ func resourceComputeInstance() *schema.Resource {
 				},
 			},
 
+			"shielded_instance_config": {
```

Comment:
I'm ngl, I like the beta naming `shielded_vm_config` a lot more but `shielded_instance_config` is the name in GA